src/styles/components/_navbar.scss

```diff
@@ -77,8 +109,15 @@
     }
   }
 
-  // TODO: for future implementation
   &__account {
+    display: flex;
+    align-items: center;
+
+    line-height: 1.52;
```

Comment:
We can remove this for now and adjust later after the desired fonts are installed.